charts/sentry/templates/sentry/metrics/deployment-sentry-metrics-consumer.yaml

```diff
@@ -89,10 +89,6 @@ spec:
           - "/tmp/health.txt"
           {{- end }}
           - "--"
```

Comment:
That line must be last in whole scope of cmd parameters
otherwise process throws error

Reply:
how reproduce error?
What is error?
I don`t have error when check PR.